etc/kayobe/stackhpc.yml

```diff
@@ -141,15 +141,11 @@ stackhpc_repo_elrepo_9_version: "{{ stackhpc_repo_distribution }}"
 
 # Kolla source repository.
 stackhpc_kolla_source_url: "https://github.com/stackhpc/kolla"
-stackhpc_kolla_source_version: "stackhpc/{{ openstack_release }}"
+stackhpc_kolla_source_version: "stackhpc/18.1.0.2"
 
 # Kolla Ansible source repository.
 stackhpc_kolla_ansible_source_url: "https://github.com/stackhpc/kolla-ansible"
-# FIXME: Waiting for
-# https://review.opendev.org/c/openstack/kolla-ansible/+/926198 to merge and
-# sync to stackhpc/2024.1
-# stackhpc_kolla_ansible_source_version: "stackhpc/{{ openstack_release }}"
```

Comment:
https://github.com/stackhpc/kolla-ansible/pull/596, available in stackhpc/18.1.0.4...